src/main/app/controller/GetController.ts

```diff
@@ -10,7 +10,11 @@ export type TranslationFn = (isDivorce: boolean) => Translations;
 
 @autobind
 export class GetController {
-  constructor(protected readonly view: string, protected readonly content: TranslationFn | Translations) {}
+  constructor(
+    protected readonly view: string,
+    protected readonly content: TranslationFn | Translations,
+    protected readonly id: string = 'page'
```

Comment:
how comes you are setting a default value here?

Reply:
👍 Good spot, I've renamed this to `stepId` to make it more obvious what it's referring to and set it to undefined by default for pages that don't need a form in bb0bf46.